aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -110,6 +128,16 @@ inline const at::Tensor& get_nested_size_tensor(const at::Tensor& tensor) {
   return get_nested_tensor_impl(tensor)->get_nested_size_tensor();
 }
 
+at::Tensor NestedTensor_clone(const at::Tensor& self, c10::optional<c10::MemoryFormat> optional_memory_format){
```

Comment:
somewhat modeled off of sparse doesn't bring any of the options with it though so probably need to fix that